ansible/aviAbsent/local.yml

```
@@ -0,0 +1,287 @@
+---
+- hosts: localhost
+  connection: local
+  gather_facts: no
+
+  collections:
+    - vmware.alb
+
+  tasks:
+    - name: Debug
+      debug:
+        msg: "{{ avi_credentials }}"
+      tags:
+        - debug
+
+    - name: Get GSLB list
+      vmware.alb.avi_api_session:
+        avi_credentials: "{{ avi_credentials | default(omit) }}"
+        api_context: "{{ avi_api_context | default(omit) }}"
+        http_method: get
+        path: gslbservice
+      register: gslbserviceResults
+      ignore_errors: yes
```

why ignore errors is set to yes?

---

This is needed in case the GSLB is not the leader in the cluster.